docs/content/en/integrations/notifications.md

```diff
@@ -16,16 +16,17 @@ and more.
 
 The following notification methods currently exist:
  - Email
- - Slack 
+ - Slack
  - Microsoft Teams
- - Alerts within DefectDojo
+ - Alerts within DefectDojo (default)
 
 You can set these notifications on a global scope (if you have
 administrator rights) or on a personal scope. For instance, an
 administrator might want notifications of all upcoming engagements sent
 to a certain Slack channel, whereas an individual user wants email
 notifications to be sent to the user\'s specified email address when a
 report has finished generating.
+Users can define notifications on a product level as well, and these settings will be applied only for selected products.
```

Comment:
There should be an empty line before this line, so that we see a new line when it's rendered.

Reply:
Applied